docs/examples/DataSet/Dataset Performance.ipynb

```diff
@@ -0,0 +1,2541 @@
+{
+ "cells": [
```

Comment:
Very nice notebook! three comments:
- could you use log scale for amount of numbers to insert?
- could you add 1.000, 10.000, perhaps 100.000 as well to the benchmarking?
- could you set the database location to smth else so that people's default `.db` files don't get suddenly filled with this test data?

Reply:
> - could you use log scale for amount of numbers to insert?
I chose a linear scale because the scaling is linear in the number of points
- could you add 1.000, 10.000, perhaps 100.000 as well to the benchmarking?
I added some more examples to the first example but not to the second one to keep the notebook runtime down
- could you set the database location to smth else so that people's default `.db` files don't get suddenly filled with this test data?
Done